tests/yam/validate/validate_desktop.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+
+# Summary: Validate agama GNOME Desktop Environment (Wayland)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "x11test";
+use Utils::Architectures qw(is_s390x is_ppc64le);
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    return if (is_s390x() || is_ppc64le());
+
+    x11_start_program "kgx";
```

Comment:
could you share what you plan to validate here? why kgx? are we missing something from non-minimal desktop?

Reply:
this is the only terminal fireup when I type term in the runner.  and here I need to run a cli in the terminal.https://openqa.suse.de/tests/17438215#step/validate_desktop/7

Reply:
you are testing the desktop itself, it is not required, that could be task for another team, we should just check presence of basic tool, pattern, packages, then you can do the same validation from the console for all the architectures.

Reply:
OK, I'd check wayland gnome pattern and related packages in select_console. Thanks